include/xrpl/protocol/ErrorCodes.h

```diff
@@ -228,26 +228,17 @@ get_error_info(error_code_i code);
 
 /** Add or update the json update to reflect the error code. */
 /** @{ */
-template <class JsonValue>
-void
-inject_error(error_code_i code, JsonValue& json)
+inline void
+inject_error(error_code_i code, Json::Value& json)
 {
     ErrorInfo const& info(get_error_info(code));
     json[jss::error] = info.token;
     json[jss::error_code] = info.code;
     json[jss::error_message] = info.message;
 }
 
-template <class JsonValue>
-void
-inject_error(int code, JsonValue& json)
-{
-    inject_error(error_code_i(code), json);
-}
-
-template <class JsonValue>
-void
-inject_error(error_code_i code, std::string const& message, JsonValue& json)
+inline void
+inject_error(error_code_i code, std::string const& message, Json::Value& json)
```

Comment:
There is probably some reason why this was `int`. Maybe someone else will be able to remember/explain why

Reply:
I assumed it was for `Json::Object` reasons